lib/ex_image_info.ex

```diff
@@ -138,6 +184,9 @@ defmodule ExImageInfo do
   def seems?(binary, :jp2), do: JP2.seems?(binary)
   def seems?(binary, :pnm), do: PNM.seems?(binary)
   def seems?(binary, :ico), do: ICO.seems?(binary)
+  def seems?(binary, :avif), do: AVIF.seems?(binary)
+  def seems?(binary, :heic), do: HEIC.seems?(binary)
+  def seems?(binary, :heif), do: HEIF.seems?(binary)
   def seems?(_, _), do: nil
```

Comment:
```suggestion
```

@warmwaffles let's continue here. I agree `func?` is usually boolean. I used the `nil` to establish sorta unknown format, but I can get rid of it (and the spec to just boolean).

So, you will get a runtime error if we don't support the format.

Anything else?

Reply:
Yea and I would also suggest changing the `type :: atom()` to be the explicit defined type `image_format()`. This way, dialyzer _should_ be able to aid people consuming it that they are about to use it wrong.

Reply:
Mainly just tightening up the API a bit is good.